std-bits/table/src/main/java/org/enso/table/excel/xssfreader/XSSFReaderSheetXMLHandler.java

```diff
@@ -0,0 +1,237 @@
+package org.enso.table.excel.xssfreader;
+
+import static org.apache.poi.xssf.usermodel.XSSFRelation.NS_SPREADSHEETML;
+
+import java.time.ZonedDateTime;
+import java.time.temporal.Temporal;
+import org.apache.poi.ss.usermodel.DateUtil;
+import org.apache.poi.xssf.model.SharedStrings;
+import org.apache.poi.xssf.usermodel.XSSFRichTextString;
+import org.enso.table.excel.ExcelUtils;
+import org.xml.sax.Attributes;
+import org.xml.sax.helpers.DefaultHandler;
+
+/** Based on the XSSFSheetXMLHandler class from Apache POI. */
+public class XSSFReaderSheetXMLHandler extends DefaultHandler {
+  private final XSSFReaderFormats styles;
+  private final SharedStrings sharedStrings;
+
+  public enum XSSDataType {
+    BOOL,
+    DATE,
+    ERROR,
+    INLINE_STRING,
+    SST_STRING,
+    NUMBER,
+    INTEGER,
+    OLE_DATE,
+    OLE_DATETIME,
+    FORMULA_STRING
+  }
+
+  // Set when V start element is seen
```

Comment:
I guess I'd need to read more about the structure to understand this, but what is "V start element"? Where should I look for documentation of these concepts?